opendj-server-legacy/resource/schema/02-config.ldif

```diff
@@ -6232,3 +6232,16 @@ objectClasses: ( 1.3.6.1.4.1.36733.2.1.2.58
         ds-cfg-bind-password $
         ds-cfg-discovery-interval )
   X-ORIGIN 'OpenDJ Directory Server' )
+objectClasses: ( 1.3.6.1.4.1.60142.2.1.2.1
```

Comment:
I would use the same OID used by Forgerock AM 7 to maintain compatibility. I checked in its schema and the OID is 1.3.6.1.4.1.36733.2.1.2.104 for ds-cfg-pbkdf2-hmac-sha256-password-storage-scheme.